package-res/plugin.spring.xml

```diff
@@ -66,4 +69,17 @@
         class="org.pentaho.reporting.platform.plugin.cache.PluginCacheManagerImpl" scope="singleton">
     <constructor-arg ref="plugin-cache-strategy"/>
   </bean>
+
+  <!-- thread pool executor -->
+  <bean id="reporting-async-thread-pool" class="org.pentaho.reporting.platform.plugin.async.PentahoAsyncExecutor" scope="singleton" >
```

Comment:
The bean ID should be IPentahoAsyncExecutor. This is the standard approach in Pentaho's spring files and we should not deviate from that standard.